### PR TITLE
Improve typography

### PR DESCRIPTION
**Pull request by wmyfelix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ If you are using Windows, please follow steps for [installing Docker Desktop on
 If you are using Linux, you will need to [install Docker Engine](https://docs.docker.com/engine/install/).
 
 `act` is currently not supported with `podman` or other container backends (it might work, but it's not guaranteed). Please see [#303](https://github.com/nektos/act/issues/303) for updates.
-
-## Homebrew
+## macOS
```
Comment (catthehacker) [inline README.md:33]: Homebrew can be used on Linux as well.

File: README.md [hunk 2]
```diff
@@ -61,16 +61,16 @@ If you are using [Scoop](https://scoop.sh/) then run:
 ```shell
 scoop install act
 ```
-
-If you are running Arch Linux, you can install the [act](https://aur.archlinux.org/packages/act/) package with your favorite package manager:
+## Linux
```
Comment (catthehacker) [inline README.md:64]: homebrew.

**Review by catthehacker — CHANGES_REQUESTED** [review]

**Comment by wmyfelix** [comment]
I'll finish the CN translations before August

**Comment by wmyfelix** [comment]
I have time to translate now, but I don’t know if it’s necessary for `act`

**Pull request closed without merging**